### PR TITLE
i18n(es): add google cloud guide

### PR DESCRIPTION
**Pull request by fredoist**
#### What kind of changes does this PR include?
- New or updated content
- Translated content

#### Description
Adds `google-cloud.md` in spanish

### CODE REVIEW

File: src/pages/es/guides/deploy/google-cloud.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Despliega tu proyecto de Astro en Google Cloud
+description: Cómo desplegar tu proyecto de Astro usando Google Cloud.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+[Google Cloud](https://cloud.google.com/) es una plataforma de alojamiento con todas las funciones, que puede ser usada para desplegar un proyecto de Astro.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/google-cloud.md:8]:
```suggestion
[Google Cloud](https://cloud.google.com/) es una plataforma integral de alojamiento, que puede ser usada para desplegar un proyecto de Astro.
```

Comment (agustinmulet) [reply src/pages/es/guides/deploy/google-cloud.md:8]: Alternativa:

```suggestion
[Google Cloud](https://cloud.google.com/) es una plataforma de alojamiento completa, que puede ser usada para desplegar un proyecto de Astro.
```

File: src/pages/es/guides/deploy/google-cloud.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Cómo desplegar
+
+Desplegar un proyecto en Google Cloud requiere algunos clics en la interfaz web. (La mayoría de estas acciones también pueden ser hechas usando la [CLI de gcloud](https://cloud.google.com/sdk/gcloud/)).
+
+### Cloud Run
+
+1. Crea un nuevo proyecto de Google Cloud, o selecciona alguno que ya tengas.
+
+2. Asegurate que la API de Cloud Run se encuentra habilitada.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/google-cloud.md:18]:
```suggestion
2. Asegúrate que la API de Cloud Run se encuentre habilitada.
```

File: src/pages/es/guides/deploy/google-cloud.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+6. Habilita el acceso público añadiendo un nuevo permiso a `allUsers` llamado `Cloud Run Invoker`.
+
+### Cloud Storage
+
+1. Crea un nuevo proyecto de Google Cloud, o selecciona alguno que ya tengas.
+
+2. Crea un nuevo contenedor dentro de [Cloud Storage](https://cloud.google.com/storage).
+
+3. Asignale un nombre y completa otros ajustes requeridos.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/google-cloud.md:34]:
```suggestion
3. Asígnale un nombre y completa otros ajustes requeridos.
```

**Review by agustinmulet — COMMENTED** [review]
Good job! Left a couple of suggestions 💪

**Comment by Yan-Thomas** [comment]
Good job @fredoist, @agustinmulet! Thank you 💜

**Pull request merged**